### PR TITLE
More Player + Aeon Classes

### PR DESCRIPTION
**Pull request by MrTyton**
Introduces new function, CurrentPlayer(), which returns the player whose turn it currently is. Did a massisve find/replace and used it as an "Easy" way to convert things to the player class system, but future work can be done to clean it up.

Added in basic aeon class. Some of the more unique aeons require additional work (yojimbo)

Added more functionality to the player class. TODOD is to complete some more additioanl functionality and add in enums for things like the spells.

TODO Is add seymour.

This change is NOT backwards compatible.

### CODE REVIEW

File: battle/boss.py [hunk 1]
```diff
@@ -799,24 +873,24 @@ def spherimorph():
...
+                elif not Lulu.active():
                     battle.main.buddy_swap(Lulu)
                 else:
-                    battle.main.defend()
+                    CurrentPlayer().defend()
             elif Lulu.is_turn():
                 if not battle.main.spheri_spell_item_ready():
                     if spell_num == 1:
-                        battle.main.ice()
+                        CurrentPlayer().cast_black_magic_spell(3)
```
Comment (jlward) [inline battle/boss.py:887]: This has gotten less easy to read. Lol. In a follow up PR, it would likely be worth converted `3` to an Enum.

Comment (MrTyton) [reply battle/boss.py:887]: Agreed, and I stated this in the PR description :P

Comment (MrTyton) [reply battle/boss.py:887]: A lot of the CurrentPlayers() can be updated to be the actual character, but this was the easiest way to do it w/ doing a mass conversion.

File: dev-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+absl-py=1.2.0
```
Comment (jlward) [inline dev-requirements.txt:1]: Might be worth addin `-r requirements.txt` to this. So we don't need dupes between "reqs required to run" and "reqs required for dev".

File: players/current_player.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+from players.tidus import Tidus
+from players.valefor import Valefor
+from players.wakka import Wakka
+from players.yojimbo import Yojimbo
+from players.yuna import Yuna
+
+
+def CurrentPlayer():
+    current_character = memory.main.get_current_turn()
+    match current_character:
```
Comment (jlward) [inline players/current_player.py:21]: You know my stance on `match`/`case`. Do with that what you will. Lol

**Review by jlward — APPROVED** [review]

**Pull request merged**